src/cogent3/parse/table.py

```diff
@@ -21,47 +21,6 @@
 __status__ = "Production"
 
 
-class ConvertFields(object):  # pragma: no cover
-    """converter for input data to Table"""
-
-    def __init__(self, conversion, by_column=True):
-        """handles conversions of columns or lines
-
-        Parameters
-        ----------
-        by_column
-            conversion will by done for each column, otherwise
-            done by entire line
-
-        """
-        super(ConvertFields, self).__init__()
-        discontinued("function", "ConvertFields", "2020.11.1")
-
-        self.conversion = conversion
-        self.by_column = by_column
-
-        self._func = self.convert_by_columns
-
-        if not self.by_column:
-            assert isinstance(
-                conversion, Callable
-            ), "conversion must be callable to convert by line"
-            self._func = self.convert_by_line
-
-    def convert_by_columns(self, line):
-        """converts each column in a line"""
-        for index, cast in self.conversion:
-            line[index] = cast(line[index])
-        return line
-
-    def convert_by_line(self, line):
-        """converts each column in a line"""
-        return self.conversion(line)
-
-    def __call__(self, *args, **kwargs):
-        return self._func(*args, **kwargs)
-
-
 def SeparatorFormatParser(
```

Comment:
there is a use of `deprecated()` for 2020.11 in this file that also needs to be removed.

Reply:
My mistake, I didn't check the full file path ... doh!

Do a project-wide search for "deprecated(" (that will identify all uses). You'll find another one specifying 2020.11 in `cogent3/util/table.py`, that also needs to be removed.